xml/operations-compute-gpu_passthrough.xml

```diff
@@ -0,0 +1,272 @@
+            your PCI device (for example Nvidia GPU):
+          </para>
+          <screen>&prompt.ardana;lspci -nn | grep -i nvidia
+            84:00.0 3D controller [0302]: NVIDIA Corporation GV100GL [Tesla V100 PCIe 16GB] [10de:1db4] (rev
+            a1)
+          </screen>
+          <para>
+            In this example, <literal>84:00.0</literal> is the address of the PCI device. The vendorID
+            is <literal>10de</literal>. The productID is <literal>1db4</literal>.
+          </para>
+        </step>
+        <step>
+          <para>
+            Confirm that the devices are available for passthrough:
+          </para>
+          <screen>&prompt.ardana;ls -ld /sys/kernel/iommu_groups/*/devices/*84:00.?/
+            drwxr-xr-x 3 root root 0 Nov 19 17:00 /sys/kernel/iommu_groups/56/devices/0000:84:00.0/
+          </screen>
+        </step>
+      </procedure>
+    </listitem>
+    <listitem>
+      <para>
+        <emphasis role="bold">Preparing &o_comp; via the input model updates</emphasis>
+      </para>
+      <para>
+        To implement the configuration you require, log into the &clm; node and update
+        the &clm; model files to enable GPU passthrough for compute nodes.
+      </para>
+      <para>
+        <emphasis role="bold">Edit servers.yml</emphasis>
+      </para>
+      <para>
+        Add the pass-through section after the definition of servers section in the yaml file.
+        Here's an example:
+      </para>
+      <screen>
+
+        ---
+        product:
+        version: 2
+
+        baremetal:
+        netmask: 255.255.255.0
+        subnet: 192.168.100.0
+
+
+        servers:
+
+        .
+        .
+        .
+        .
+
+        pass-through:
+          servers:
+            - id: compute-0001
+              data:
+                gpu:
+                  - vendor_id: 10de
+                    product_id: 1db4
+                    bus_address: 0000:84:00.0
+                    pf_mode: type-PCI
+                    name: a1
+                  - vendor_id: 10de
+                    product_id: 1db4
+                    bus_address: 0000:85:00.0
+                    pf_mode: type-PCI
+                    name: b1
+            - id: compute-0008
+              data:
+                gpu:
+                  - vendor_id: 10de
+                    product_id: 1db4
+                    pf_mode: type-PCI
+                    name: c1
+      </screen>
+      <para>
+        To make the edits,
+      </para>
+      <procedure>
+        <step>
+          <para>
+            Check out the site branch of the local git repository and change to the
+            correct directory:
+          </para>
+          <screen>
+            &prompt.ardana;git checkout site
+            &prompt.ardana;cd ~/openstack/my_cloud/definition/data/
+          </screen>
+        </step>
+        <step>
+          <para>
+            Open servers.yml with vim or another editor and make the necessary changes.
```

Comment:
Technically this could be any file name if the customer has re-structured the model contents... So being pedantic the wording might be better written as "Open the file containing the servers list, e.g. servers.yml, ..."

Reply:
Updated the wording